contracts/Vault.vy

```diff
@@ -314,13 +314,19 @@ def _issueSharesForAmount(_to: address, _amount: uint256) -> uint256:
     return shares
 
 
-@internal
-def _depositFrom(_sender: address, _recipient: address, _amount: uint256) -> uint256:
+@external
+def deposit(_recipient: address = msg.sender, _amount: uint256 = MAX_UINT256) -> uint256:
```

Comment:
nice! I was playing with this yesterday and I didn't understand why this signature doesn't allow:
```
deposit(_amount=1234)
```

Somehow vyper is not generating a method for `deposit(_amount)` which would be when `_recipient` is the default.

Reply:
ah yes, I see your problem now @Macarse, web3py automagically finds the correct signature based on the type of all the arguments (throwing if there's multiple matches). You don't use kwargs directly for this (e.g. `_amount=12354` won't work)

Reply:
@chriswessels swap the order of these arguments so that `deposit(1234)` can be called to deposit the amount mentioned, it is definitely expected to call with this more often than with delegated amounts

Reply:
@fubuloubu Done!